src/parser/jobs/blu/modules/MoonFlute.tsx

```diff
@@ -186,31 +242,50 @@ export class MoonFlute extends BuffWindow {
 
 	private adjustExpectedActionOutcome(window: HistoryEntry<EvaluatedAction[]>, trackedActions: TrackedActionGroup) {
 		const finalStingUsed = this.finalStingsUsedInWindow(window)
-		if (finalStingUsed === 0) {
-			// Final Sting not used here
-			if (trackedActions.actions.length !== 1) {
-				// Default handling:
-				return
-			}
-			const trackedActionId = trackedActions.actions[0].id
-			if (trackedActionId === this.data.actions.TRIPLE_TRIDENT.id) {
-				// Using TT on cooldown, on a long enough timeline, can be a DPS gain over
-				// holding it for a MF window, particularly for SpS builds.
-				// So let's be understanding -- We only dock points if they were in a
-				// Moon Flute window, had TT available, and didn't use it.
-				if (!this.windowMissedTT.has(window.start)) {
-					// TT was either used, or wasn't available during the window.
-					// Either way, this check will either be a Positive or a Neutral,
-					// never a negative.
+		if (finalStingUsed !== 0) {
+			// Final Sting used, so don't dock any points for missed actions:
+			return neutralOrPositiveOutcome
+		}
+
+		// Final Sting not used here
+		if (this.breathOfMagicApplier) {
+			// It's the Breath of Magic applier!  They may be doing an off-minute window to
+			// reapply BoM, and we should not dock them any points for that.
+			// ...but how can we tell if it's an off-minute window?
+			// Yeah, that's a good question.  Hopefully someone will have a better
```

Comment:
This isn't the silliest thing I've ever seen done for guesswork! You should see the nonsense that is DNC's Gauge code :)

Reply:
What about doing a check against last Nightbloom use < 120s and Breath of Magic.  That way you don't get a false positive against someone not bringing Nightbloom at all.

Reply:
Latest patch has a reimplementation of @xiashtra's heuristic using the Nightbloom cooldown as the heuristic -- if they didn't take Nightbloom then everything is strict.  If they took Nightbloom, and a moon flute happens while Nightbloom was still on cooldown, AND they used BoM, we assume it's an odd-minute and relax the check.

As a TODO for later, the odd-minute check should still disallow Surpanakha unless it is exactly 0 or 4. Anything in-between in lost potency.

I also reworded the suggestions from the module slightly, both for clarity and to point out that the odd-minute MF is excempt from some checks.

Reply:
...also, oh gosh, I just looked at DNC's Gauge calculations.  BLU has it good.